### PR TITLE
Feature/support arm 64 epmdj 10882

### PR DESCRIPTION
**Pull request by bodyangug**

### CODE REVIEW

File: logging/src/nativeMain/kotlin/com/epam/drill/logging/FileOutputAppender.kt [hunk 1]
```diff
@@ -28,20 +28,23 @@ import platform.posix.open
...
     private val output = Output(open(filename, openFlags, openMode))
 
-    override fun trace(message: Any?) = append(message).getOrDefault(Unit)
-    override fun debug(message: Any?) = append(message).getOrDefault(Unit)
-    override fun info(message: Any?) = append(message).getOrDefault(Unit)
-    override fun warn(message: Any?) = append(message).getOrDefault(Unit)
-    override fun error(message: Any?) = append(message).getOrDefault(Unit)
 
-    fun close() = output.close()
+    override fun trace(loggerName: String, message: String) = append(message).getOrDefault(Unit)
```
Comment (RomanDavlyatshin) [inline logging/src/nativeMain/kotlin/com/epam/drill/logging/FileOutputAppender.kt:39]: why do we have to pass logger name as parameter - isn't it should be defined by logging configuration?

**Review by RomanDavlyatshin — COMMENTED** [review]
see comment above

File: logging/src/nativeMain/kotlin/com/epam/drill/logging/FileOutputAppender.kt (hunk 1 above)
Comment (bodyangug) [reply logging/src/nativeMain/kotlin/com/epam/drill/logging/FileOutputAppender.kt:39]: Due to the higher Kotlin version, we have new method declarations for the Appender.

File: agent-config/build.gradle.kts [hunk 2]
```diff
@@ -23,15 +23,20 @@ repositories {
 }
 
 kotlin {
-    targets {
```
[the remark this answers is not in the document]
Comment (bodyangug) [reply agent-config/build.gradle.kts:26]: due to new kotlin versions: "The presets API is deprecated and will be removed in future releases."

File: agent-instrumentation/build.gradle.kts [hunk 3]
```diff
@@ -41,22 +41,27 @@ kotlin {
             compilation = compilations["intTest"]
         }
     }
-    targets {
```
[the remark this answers is not in the document]
Comment (bodyangug) [reply agent-instrumentation/build.gradle.kts:44]: due to new kotlin versions: "The presets API is deprecated and will be removed in future releases."

File: common/build.gradle.kts [hunk 4]
```diff
@@ -23,15 +23,20 @@ repositories {
 }
 
 kotlin {
-    targets {
```
[the remark this answers is not in the document]
Comment (bodyangug) [reply common/build.gradle.kts:26]: due to new kotlin versions: "The presets API is deprecated and will be removed in future releases."

File: logging-native/src/nativeMain/kotlin/mu/Appender.kt [hunk 5]
```diff
@@ -1,9 +1,11 @@
 package mu
 
 public interface Appender {
-    public fun trace(message: Any?)
-    public fun debug(message: Any?)
-    public fun info(message: Any?)
-    public fun warn(message: Any?)
-    public fun error(message: Any?)
+    public val includePrefix: Boolean
```
[the remark this answers is not in the document]
Comment (bodyangug) [reply logging-native/src/nativeMain/kotlin/mu/Appender.kt]: logging-native module was removed

**Pull request merged**